src/js/sources.js

```diff
@@ -25,14 +23,11 @@ export const sources = {
   ytcDeletions: writable(null),
 };
 
-/** @type {(id: String) => Boolean} */
-const userBlacklisted = id => channelFilters.get(id).blacklist;
-
 /** @type {(msg: Message) => Boolean} */
 const isWhitelisted = msg => textWhitelisted(msg.text) || authorWhitelisted(msg.author);
 
 /** @type {(msg: Message) => Boolean} */
-const isBlacklisted = msg => textBlacklisted(msg.text) || userBlacklisted(msg.authorId) || authorBlacklisted(msg.author);
+const isBlacklisted = msg => textBlacklisted(msg.text) || authorBlacklisted(msg.author);
```

Comment:
Was the `userBlacklisted` logic moved/removed in a previous commit? Right now banning YT users does not work.

Reply:
Fixed in 5f7668f.